apps/website/blog/2024-11-20-maci-platform.md

```diff
@@ -0,0 +1,93 @@
+
+- Dashboard for contract deployment and round configuration
+- Allowing multiple rounds to live within one MACI Platform instance
+- Payout disbursement via the interface by either the voters, applicants, or operators
+- Full integration with [MACI’s coordinator service](https://github.com/privacy-scaling-explorations/maci-platform/issues/278) to allow finalization of rounds within the interface
+- Continued integration with different gatekeepers like [Gitcoin Passport](https://docs.passport.xyz/)
+
+Furthermore, we plan to experiment with AA (account abstraction) to enable smooth experiences for both users and organisers. For users, we have implemented an end-to-end flow using an embedded wallet and paymaster as part of PSE’s internal [hackathon](https://github.com/privacy-scaling-explorations/hacking-pse/tree/main/packages/interface). For organizers we will be using a smart account and [session keys](https://docs.zerodev.app/sdk/advanced/session-keys) to safely interact with the coordinator service and interact with MACI contracts.
+
+### Ecosystem Harmony
+
+Projects like Gitcoin, Dorahacks, Snapshot, Devfolio, or Jokerace all seem to offer very similar voting, polling, and quadratic funding solutions. So what makes MACI different?
+
+At MACI, our goal isn’t to compete, but to help push the ecosystem toward using safer, more private, and collusion-resistant voting mechanisms. We're actively working with many of the above projects to integrate the MACI protocol, ensuring that the benefits of secure voting can be used where they’re needed most. MACI Platform presents itself as just one more quadratic funding platform for communities to choose from.
+
+It's important to note that while the anti-collusion properties of MACI provide higher security guarantees, they do come with tradeoffs— particularly in terms of user experience (UX), cost, and simplicity. The MACI team is continuously working to address these tradeoffs, but for certain use cases, these tradeoffs ultimately might not be worth it. For example, the encryption of votes at the end of a round may not actually be desirable or aligned with the spirit or values of any specific community. MACI Platform aims to provide privacy solutions where they’re needed the most, and to augment and support existing voting platforms.
+
+### How to get involved
+
+If you are a community event organizer and want to run a voting or funding round using MACI Platform, please reach out to us using this [form](https://esp.ethereum.foundation/pse-sponsorships/apply). If you are a developer and want to contribute to the code then head over to our [GitHub repo](https://github.com/privacy-scaling-explorations/maci-platform/issues) and take a look at the “good first issues” tags, or open a new one that you think it’s beneficial and you would like to work on.
+
+Also, stay tuned for recurring voting rounds on https://maci.vote and our participation in hackathon as sponsors.
```

Comment:
the website is not working now, should we separate the vercel for running `maci.vote` and other specific rounds?